src/aria/widgets/form/CheckBox.js

```diff
@@ -104,8 +96,8 @@ module.exports = Aria.classDefinition({
          */
         _checkCfgConsistency : function () {
             // for the simpleHTML case, tab index is set directly on the input
```

Comment:
This comment should be updated. Now the tab index is always set directly on the input.